typings/global.d.ts

```diff
@@ -11,3 +11,5 @@ declare interface System {
   import<T = any>(module: string): Promise<T>
 }
 declare var System: System;
+
+declare module '*';
```

Comment:
Imo we should avoid doing things like that. If you import some module that doesn't have it's own typings at least use it's name instead of asterisk.